docs/en/rst/api/core/v1/bugzilla.rst

```diff
@@ -300,3 +300,32 @@ name             type    description
 ===============  ======  ====================================================
 last_audit_time  string  The maximum of the at_time from the audit_log.
 ===============  ======  ====================================================
+
+Job Queue Status
+----------------
+
+Reports the status of the job queue.
+
+**Request**
+
+.. code-block:: text
+
+   GET /rest/jobqueue_status
+
+This method requires an authenticated user. 
+
+**Response**
+
+.. code-block:: js
+
+   {
+     "total": 12,
+     "errors": 0
+   }
+
+===============  =======  ====================================================
+name             type     description
+===============  =======  ====================================================
+total            integer  The total number of jobs in the job queue.
+errors           integer  The number of errors produced by jobs in the queue.
```

Comment:
- I think this is right? Or is it 'the number of jobs that have errored out'? The db query makes it seem like there can be more than one error per job.